### ISSUE TITLE
slow collecting when install via pip

### ISSUE THREAD

**Issue by DeoLeung** [pydata/bottleneck#192]
I'm experiencing slow installation via pip
```bash
pip install bottleneck
Collecting bottleneck==1.2.1 (from -r /tmp/requirements.txt (line 6))
  Downloading http://xxxx/+f/7fb/e5f1194b2717d/Bottleneck-1.2.1.tar.gz (105kB)
# after this showed up it stuck for quite a few minute
```
any hint on what happens?

I'm using a docker `python:3.6.6` container to install packages from a `devpi` container

**Comment by DeoLeung** [comment]
after playing around I figure out that it's due to `numpy` and `bottleneck` written in the same requirements.txt file

which when our internal network can't access `pypi`, it stucks, if we install numpy first, then bottleneck, it's fine

maybe someone can fix the setup.py ( sorry I can't figure out where the network check happens)

**Comment by qwhelan** [comment]
@DeoLeung I realize it's been a while, but are you still able to reproduce this issue? There have been a number of changes in master that would potentially fix this issue, but I'd like to confirm if at all possible.

**Comment by qwhelan** [comment]
Closing as I believe this is resolved by the 1.3.0 release. Please feel free to re-open if that is not the case.
